ensembl/conf/ini-files/oryctolagus_cuniculus.ini

```diff
@@ -65,8 +65,10 @@ ENSEMBL_SEARCH_IDXS = [ Domain Family Gene GenomicAlignment Marker Sequence ]
 
 SEARCH_TEXT       = glycoprotein
 
+VARIATION_PARAM   = 2:6565980:G_C:PRJEB27278;vf=2:6565980:G_C:PRJEB27278
+VARIATION_TEXT    = 2:6565980:G_C:PRJEB27278
+
 ENSEMBL_SOUND     = Eh, what's up, doc?
 VEP_HGVS          = ENSOCUT00000014253.2:c.4C>A\nENSOCUT00000008071.3:c.168+16T>C\nENSOCUT00000016986.3:c.5delA
```

Comment:
Could you please regenerate the VEP examples. The transcript versions are outdated. It is best to also make sure that the VEP examples are up-to-date for new VCF species. I already updated the documentation. Thank you!